telemetry/definitions/commonDefinitions.json

```diff
@@ -5959,6 +5964,36 @@
                 }
             ]
         },
+        {
+            "name": "toolkit_componentInit",
+            "description": "The Toolkit has completed initialization for the specified component.",
+            "metadata": [
+                {
+                    "type": "attempts",
+                    "required": false
+                },
+                { 
+                    "type": "duration",
+                    "required": false
+                },
+                { 
+                    "type": "errorCode",
```

Comment:
Reason, errorCode is already part of our error metadata optional fields and does not need to be specified here

Reply:
ah thanks for catching that - i'll remove those